config/config-example.js

```diff
@@ -493,6 +493,7 @@ exports.grouplist = [
 		inherit: '@',
```

Comment:
I wonder whether instead of doing this, shouldn't this be changed to inherit from `*` instead of `@`. @Zarel?

Reply:
I don't want bots part of the regular inheritance chain, even if ROs are supposed to be a strict superset. I've always modeled them as a fork.